### PR TITLE
Fore-and-back ground sync

### PR DESCRIPTION
**Pull request by AloeareV**
Draft as it's built atop #180, and I want to write a test that uses it before merging (I'm not worried about getting it in too quickly, as we're planning on a more stable release schedule, yes?)

### CODE REVIEW

File: android/app/src/main/res/xml/network_security_config.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<network-security-config>
+    <domain-config cleartextTrafficPermitted="true">
+        <domain includeSubdomains="true">10.0.2.2</domain>
```
Comment (zancas) [inline android/app/src/main/res/xml/network_security_config.xml:4]: Why is 10.0.2.2 permitted?

Comment (AloeareV) [reply android/app/src/main/res/xml/network_security_config.xml:4]: https://developer.android.com/studio/run/emulator-networking

"The virtual router for each instance manages the 10.0.2/24 network address space. All addresses managed by the router are in the form of 10.0.2.xx, where xx is a number. Addresses within this space are pre-allocated by the emulator or router as follows:

...

10.0.2.2 | Special alias to your host loopback interface (127.0.0.1 on your development machine)"

**Comment by Oscar-Pepper** [comment]
adding ref to original background sync PR #160

**Comment by Oscar-Pepper** [comment]
continued in #322

**Pull request closed without merging**